lib/model/entity/bibliotheek/BoekExemplaar.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+
+namespace CsrDelft\model\entity\bibliotheek;
+
+
+use CsrDelft\model\bibliotheek\BoekModel;
+use CsrDelft\model\security\LoginModel;
+use CsrDelft\Orm\Entity\PersistentEntity;
+use CsrDelft\Orm\Entity\T;
+
+class BoekExemplaar extends PersistentEntity {
```

Comment:
Volgorde van deze klasse is een beetje raar. Beter eerst fields, dan functions en dan protected static fields.